### ISSUE TITLE
[FEAT]: Silence skip feature

### ISSUE THREAD

**Issue by song2daodao** [nova-video-player/aos-AVP#841]
### Description

Just like the Justplayer or Newpipe. And they both base on Exoplayer. 

Exo provides great feature called [silence skip](https://github.com/google/ExoPlayer/blob/release-v2/library/core/src/main/java/com/google/android/exoplayer2/audio/SilenceSkippingAudioProcessor.java)

And if video can play directly when you click first time that would be great. I have to click twice on each video, a little bit annoying.
The skip silence feature is very useful for watching online lessons videos, please consider to adding this feature, thanks!!!

**Comment by PacoBell** [comment]
Yes, please implement this, IMHO, critical feature. The other players mentioned either have playback speed controls >2.0x or silence skipping, but not both. Nova could be the first Android video player to do so.
